cmake/modules/hunter_download.cmake

```diff
@@ -324,14 +326,29 @@ function(hunter_download)
         "Internal dependencies ID: ${HUNTER_PACKAGE_INTERNAL_DEPS_ID}"
     )
   endif()
-
+  
+  set(_hunter_schemes_search_dirs "")
+  
   set(
-      download_scheme
-      "${HUNTER_SELF}/cmake/schemes/${HUNTER_DOWNLOAD_SCHEME}.cmake.in"
+    download_scheme
```

Comment:
Indend is 4 spaces here: http://0.readthedocs.org/en/latest/cmake.html#indentation